### PR TITLE
less backoff timeout try 2

### PR DESCRIPTION
**Pull request by CheSema**
reverts https://github.com/ClickHouse/ClickHouse/pull/65148
original PR is https://github.com/ClickHouse/ClickHouse/pull/63857

### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
S3. reduce retires time for queries, increase retries count for backups. 8.5 minutes and 100 retires for queries, 1.2 hours and 1000 retries for backup restore.

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: Normal Builds
- [ ]  Allow: Special Builds
- [ ]  Allow: All NOT Required Checks
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Do not test
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 13e00d4f8b621c800b286e383b20765e669d5163 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/65232/13e00d4f8b621c800b286e383b20765e669d5163/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>A Sync</td><td>If it fails, ask a maintainer for help</td><td>⏳ pending</td></tr>
<tbody>
</table>
<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>Checks correctness of the PR's body</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Comment by CheSema** [comment]
cc @Algunenano 
I fixed the setting history. CI is green. Lets try again.

File: src/Core/SettingsChangesHistory.h [hunk 1]
```diff
@@ -111,6 +111,7 @@ static const std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges
               {"output_format_csv_serialize_tuple_into_separate_columns", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_deserialize_separate_columns_into_tuple", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_try_infer_strings_from_quoted_tuples", true, true, "A new way of how interpret tuples in CSV format was added."},
+              {"backup_restore_s3_retry_attempts", 0, 1000, "A new setting."},
```
Comment (Algunenano) [inline src/Core/SettingsChangesHistory.h:114]: 24.6 is released. This will need to be changed to 24.7

Comment (CheSema) [reply src/Core/SettingsChangesHistory.h:114]: Do not understand what you mean.

Comment (CheSema) [reply src/Core/SettingsChangesHistory.h:114]: Oh. I see it is rotten already because it was not merged on time.

Comment (CheSema) [reply src/Core/SettingsChangesHistory.h:114]: I already curious is it even possible to merge something without overriding CI block.

**Review by Algunenano — APPROVED** [review]

**Pull request merged**